test/e2e/framework/framework.go

```diff
@@ -141,6 +142,15 @@ func (f *Framework) CommonLabels() map[string]string {
 	}
 }
 
+func (f *Framework) CurrentSpecAnnotations() map[string]string {
```

Comment:
seems more private but maybe just inlining it into setupNamespace would be best to avoid having to merge maps when we want to add an annotation

Reply:
I did it this way so that it could be repurposed, but I don't really see the need for it now. Changed.